### PR TITLE
Added some minor modes documentation

### PR DESCRIPTION
**Pull request by ducky427**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,14 +72,21 @@ Other recommended settings
 
 Since `hy-mode` is a lisp like mode, it can use the goodness from some
 minor modes used for lisp such as
-[smartparens](https://github.com/Fuco1/smartparens) or paredit minor
+[smartparens](https://github.com/Fuco1/smartparens), 
+[RainbowDelimiters](https://github.com/Fanael/rainbow-delimiters) or paredit minor
```
Comment (theanalyst) [inline README.md]: could you move this line ie. rainbow delim above.. bascially the line should read smartparens or paredit minor modes...

**Comment by ducky427** [comment]
I've removed rainbow delimiter from the readme.

imho, I find it really useful for reading lisp code.

**Comment by theanalyst** [comment]
@ducky427 i didn't mean to remove the ref. to rainbow delimiter.. what I meant was put rainbow-delims before smartparens.. since the sentence was reading smartparens or paredit which support navigation

**Comment by ducky427** [comment]
Ah. my bad. let me do it again.

**Comment by ducky427** [comment]
Added rainbow delimiters back. 

btw, I am a clojure user and so use the excellent [clojure-mode](https://github.com/clojure-emacs/clojure-mode).

**Comment by theanalyst** [comment]
sure, but shouldn't the rainbow-delims link to rainbow-delims github ;)

**Comment by ducky427** [comment]
doh!

**Comment by theanalyst** [comment]
thanks ! merged

**Comment by ducky427** [comment]
cheers!

**Pull request merged**